pkg/relay/ethereum/client.go

```diff
@@ -23,6 +23,7 @@ func (chain *Chain) CallOpts(ctx context.Context, height int64) *bind.CallOpts {
 }
 
 func (chain *Chain) TxOpts(ctx context.Context) *bind.TransactOpts {
+	logger := chain.GetChainLogger()
```

Comment:
Move this into the closure.

Reply:
Move to closure
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/441f8e7d43fabe7e6f4dc54d2b20933d8408725c